definitions/pipelines/somatic_exome_cle_gathered.cwl

```diff
@@ -38,6 +38,20 @@ inputs:
         type: File
     target_intervals:
         type: File
+        label: "target_intervals: interval_list file of targets used in the sequencing experiment"
+        doc: |
+          target_intervals is an interval_list corresponding to the targets for the sequencing reagent.
+          These are essentially coordinates for regions designed probes for in the reagent.
```

Comment:
Looks like this sentence was wholly removed from many of these inputs, but it's still here (and in three other places).